scss/details.scss

```diff
@@ -26,6 +26,22 @@
   box-shadow: 4px 4px 10px 0 rgba(0,0,0,0.5);
 }
 
+.img-container {
+  position: relative;
+
+  &:after {
+    @include transform(rotate(20deg));
+    background: url('../img/badge.png') no-repeat;
+    background-size: contain;
```

Comment:
It's ok, but be careful with this, it's not supported in all browsers (https://caniuse.com/#search=contain)